keyboards/fleuron/keymaps/default/keymap.c

```diff
@@ -0,0 +1,168 @@
+  KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,  KC_P1,  KC_P2,    KC_P3,   KC_PPLS, \
+  RGB_MOD, KC_LCTL, KC_LALT, KC_LGUI, LOWER,  KC_SPC,  KC_SPC,   RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, KC_P0,  DZ,       KC_PDOT, KC_PENT \
+),
+
+[_LOWER] = KEYMAP(
+  /* Lower
+   * ,---------------------------------------------------------------------------------------------------------------.
+   * |RESET |      |      |      |      |      |      |      |      |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |      |      |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |      |      |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+-------------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      | _    | +    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      | {    | }    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |RGBtog|      |      |      |      |      |      |      | Home | PgUp | PgDn | End  |      |      |      |      |
+   * `---------------------------------------------------------------------------------------------------------------'
+   */
+  RESET,    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  _______,  _______,  _______,  _______,  _______,  KC_UNDS,  KC_PLUS,  _______,  _______,  _______,  _______,  _______,  _______,  _______, \
+  _______,  _______,  _______,  _______,  _______,  _______,  _______,  KC_LCBR,  KC_RCBR,  _______,  _______,  _______,  _______,  _______,  _______,  _______, \
+  RGB_TOG,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  KC_HOME,  KC_PGDN,  KC_PGUP,  KC_END,   _______,  _______,  _______,  _______ \
+),
+
+[_RAISE] = KEYMAP(
```

Comment:
This should be changed to `LAYOUT_ortho_6x16` (needs to match the define in the h file for the keyboard).

Reply:
Done :)